src/App.vue

```diff
@@ -731,18 +752,29 @@ export default {
       }
     },
 
-    storeDirectors (response: any): void {
-      const directors = response?.data?.directors
-      if (directors) {
-        const directorsSorted = directors.sort(this.fieldSorter(['lastName', 'firstName', 'middleName']))
-        for (var i = 0; i < directorsSorted.length; i++) {
-          directorsSorted[i].id = i + 1
-          directorsSorted[i].isNew = false
-          directorsSorted[i].isDirectorActive = true
+    storeParties (response: any): void {
```

Comment:
Updated to handle both custodians and directors. 
This also filters our some instances where completing parties were returned as duplicates of the directors in the directors request.